website/docs/getting-started/for-designers.md

```diff
@@ -208,4 +208,4 @@ Explore more content about libraries, components, and styles directly from Figma
 
 ## Support
 
-If you have questions or need assistance using Helios libraries, <LinkTo class="doc-link-generic" @route="show" @model="about/support">reach out for support</LinkTo>.
+ <LinkTo class="doc-link-generic" @route="show" @model="about/support">Contact the Helios Design System team</LinkTo> for support if you have questions or need assistance using Helios libraries.
```

Comment:
Trimmed an extra space as well (probably doesn't matter to the compiler though?)

```suggestion
<LinkTo class="doc-link-generic" @route="show" @model="about/support">Contact the Helios Design Systems Team</LinkTo> for support if you have questions or need assistance using Helios libraries.
```